rest-service/pom.xml

```diff
@@ -52,67 +88,24 @@
 			<groupId>org.jboss.spec.javax.servlet</groupId>
 			<artifactId>jboss-servlet-api_3.1_spec</artifactId>
 		</dependency>
+        <dependency>
+            <groupId>org.jboss.spec.javax.xml.bind</groupId>
+            <artifactId>jboss-jaxb-api_2.3_spec</artifactId>
+        </dependency>
 		<dependency>
 			<groupId>io.swagger</groupId>
```

Comment:
adding and removing new dependencies for basepom change sure recipe for failure, unless these were transitive ones from before